protocol/src/authentication.rs

```diff
@@ -24,7 +24,7 @@ use protobuf::ProtobufEnum as ProtobufEnum_imported_for_functions;
 
 /// Generated files are compatible only with the same version
 /// of protobuf runtime.
-const _PROTOBUF_VERSION_CHECK: () = ::protobuf::VERSION_2_8_1;
```

Comment:
Hm, I'm not sure what's happened here...

Looks like maybe you built with an incorrect `Cargo.lock`. Have you tried building since getting the lock file correct?

It doesn't look like the protobuf version changed in the lock file so a newer version shouldn't have been used here as far as I understand it.

Reply:
Looks like this is the `Cargo.lock` from master..